### PR TITLE
Use thoth.common to setup logging

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [ ] Yes
- [x] No

Depends-On: #3

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -9,6 +9,8 @@ verify_ssl = true
 thoth-python = "*"
 thoth-storages = "*"
 faust = "*"
+kafka = "*"
```
Comment (KPostOffice) [inline Pipfile]: No longer needed

Comment (fridex) [reply Pipfile]: Removed.

File: messages/hash_mismatch.py [hunk 2]
```diff
@@ -18,16 +18,13 @@
...
-import os
-import json
 import logging
 
 import faust
 
 from .message_base import MessageBase
 
-_LOGGER = logging.getLogger(__name__)
+_LOGGER = logging.getLogger("thoth.update_job.messages")
```
Comment (goern) [inline messages/hash_mismatch.py]: why is this "update_job" and the other are "package_update" ?

Comment (fridex) [reply messages/hash_mismatch.py]: Good catch, fixed.

**Comment by harshad16** [comment]
@fridex This LGTM, if you can rebase and push. i will approve this. 
thanks :)

**Comment by fridex** [comment]
> @fridex This LGTM, if you can rebase and push. i will approve this.
> thanks :)

Rebased, PTAL.

Thanks! :)

**Comment by harshad16** [comment]
i can see gating happening.

**Comment by goern** [comment]
is this still valid?

**Comment by fridex** [comment]
Not anymore, it looks like all the parts use `init_logging` from thoth-common.

**Pull request closed without merging**